packages/clerk-js/src/utils/queryStateParams.ts

```diff
@@ -0,0 +1,51 @@
+import { encodeB64, getClerkQueryParam } from '../utils';
+
+export const buildUrl = ({ base, path }: { base: string; path: string | undefined }) => {
```

Comment:
🔧 I see that we already have an exported `buildURL` from utils. We could rename this.

Reply:
@anagstef the util you are referring to is doing another thing. It can't be used for our case and I think it should not be refactored to do many things.

Reply:
@anagstef changed the naming here